### ISSUE TITLE
Which features of libwebsockets are used?

### ISSUE THREAD

**Issue by karlp** [tsl0922/ttyd#452]
Hi,  I'm trying to cleanup the libwebsockets packaging for openwrt, which libwebsockets features do you need?  libuv, but what else?

**Comment by karlp** [comment]
ALso, I've tried building against the forthcoming 4.1 release, and it fails here: 
```
[ 16%] ESC[32mBuilding C object CMakeFiles/ttyd.dir/src/server.c.oESC[0m
src/server.c: In function 'main':
src/server.c:455:7: error: 'struct lws_context_creation_info' has no member named 'ws_ping_pong_interval'
   info.ws_ping_pong_interval = 5;
       ^
```

**Comment by tsl0922** [comment]
> src/server.c:455:7: error: 'struct lws_context_creation_info' has no member named 'ws_ping_pong_interval'

It's already fixed on master (5902726732ff04db6edf57f2148cf8cb166fb7ee), but not released yet.

libuv is required, unix sock and ipv6 are optional.

**Comment by karlp** [comment]
Ok, thanks!
